lib/broccoli/angular-broccoli-compass.js

```diff
@@ -0,0 +1,66 @@
+/* jshint node: true, esversion: 6 */
+'use strict';
+
+let sass;
+let compass;
+try {
+  try {
+    sass = require('sass');
+    compass = require('compass');
+  } catch (e) {
+    sass    = require(`${process.env.PROJECT_ROOT}/node_modules/node-sass`);
```

Comment:
If `${process.env.PROJECT_ROOT}/node_modules/node-sass` exists, wouldn't `require('sass')` have picked it up ?